app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -233,6 +237,10 @@ public LeakTool getLeakTool() {
         .addLogger(new ConsoleLogger());
     Logger.setDBG(ToolboxManager.isDebug(getDefaultSharedPreferences()) || BuildConfig.DEBUG);
 
+    Single.fromCallable(() -> aptoideMd5Manager.calculateMd5Sum())
+        .subscribeOn(Schedulers.io())
+        .observeOn(AndroidSchedulers.mainThread());
```

Comment:
You were testing this but... there's no subscription... It never did anything 🤣🤣🤣🤣🤣